### DISCUSSION TITLE
What is the suggested method to access flash in a view or component?

### DISCUSSION THREAD

**Discussion by rmontgomery429** [yippee-fun/phlex#601] [Help] [answered]
I'm not seeing anyone else struggling with this so maybe I have it upside down but I can't see to get the `flash` inside of any Layout, View or Component. Has anyone found a way to work with things like flash or session?

**Comment by joeldrapper** [comment, marked as answer]
@rmontgomery429 you should be able to use `helpers.flash`, but we should add an adapter module for this so you can include `Phlex::Rails::Helpers::Flash` and then just use `flash`. I don't know how I missed this one.

**Reply by rmontgomery429** [reply]
That would be great but for now `helpers.flash` totally works. Thank you!
